src/XRP/serializer.ts

```diff
@@ -37,9 +38,9 @@ class Serializer {
   /**
    * Convert a Transaction to a JSON representation.
    *
-   * @param {proto.Transaction} transaction A Transaction to convert.
-   * @param signature An optional hex encoded signature to include in the transaction.
-   * @returns {Object} The Transaction as JSON.
+   * @param transaction - A Transaction to convert.
```

Comment:
The TSDoc ESLint plugin bans the use of types in the JSDoc comments, because we're using TypeScript and we already have that information available to us.